source/MaterialXView/Viewer.cpp

```diff
@@ -279,6 +279,10 @@ Viewer::Viewer(const std::string& materialFilename,
     _genContextMdl.getOptions().targetColorSpaceOverride = "lin_rec709";
     _genContextMdl.getOptions().fileTextureVerticalFlip = false;
 #endif
+#if MATERIALX_BUILD_GEN_ARNOLD
```

Comment:
Also fix up MTLXViewer as it was not setting up units properly for Arnold codegen save.